app/models/ncr/work_order.rb

```diff
@@ -114,8 +114,13 @@ def approving_official
       self.approvers.first
     end
 
-    def approving_official_email_address
-      approving_official ? approving_official.email_address : self.system_approver_emails.first
+    # the highest approver on the stack, pending preferred if status indicates
```

Comment:
what do you think about not having a comment here? I think this method's tests make it clear why it works this way, and having the comment could lead to more complications down the line.

For more details on this line of thinking, this is a killer blog post on why code comments can be tricky https://robots.thoughtbot.com/letting-your-code-speak-for-itself by the one and only @adarsh